tests/bluetooth/bsim_bt/bsim_test_eatt/src/main_collision.c

```diff
@@ -12,8 +12,16 @@ static void test_peripheral_main(void)
 {
 	int err;
 
+	backchannel_init();
+
 	peripheral_setup_and_connect();
 
+	/*
+	 * we need to sync with peer to ensure that we get collisions
+	 */
+	backchannel_sync_send();
+	backchannel_sync_wait();
+
```

Comment:
This is test procedure change, changes should be separate PR and merged before this PR. And these changes should pass CI before and after switch to refactored LLCP.

Reply:
> This is test procedure change, changes should be separate PR and merged before this PR. And these changes should pass CI before and after switch to refactored LLCP.

See PR #48859